### PR TITLE
Add lightweight field_number

### PR DESCRIPTION
**Pull request by rachel-fenichel**

### CODE REVIEW

File: core/field_number.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  Blockly.FieldNumber.superClass_.constructor.call(this, text,
+      opt_validator);
+};
+goog.inherits(Blockly.FieldNumber, Blockly.FieldTextInput);
+
+/**
+ * Return the current value.
+ * @return {number} Current value as a number
+ */
+Blockly.FieldColour.prototype.getValue = function() {
```
Comment (NeilFraser) [inline core/field_number.js]: FieldColour?

File: core/field_number.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+ */
+Blockly.FieldNumber = function(text, opt_validator) {
+  Blockly.FieldNumber.superClass_.constructor.call(this, text,
+      opt_validator);
+};
+goog.inherits(Blockly.FieldNumber, Blockly.FieldTextInput);
+
+/**
+ * Return the current value.
+ * @return {number} Current value as a number
```
Comment (NeilFraser) [inline core/field_number.js]: End with period.

**Comment by NeilFraser** [comment]
Change math_number and controls_repeat blocks to use this (that way the code is actually used somewhere in Blockly).  Then go ahead and commit.

**Comment by rachel-fenichel** [comment]
Looks like this works for xml -> blocks -> json, but fails in fieldToDom when trying to actually render it.  Will take a look.

**Pull request merged**